sections/staking/components/StakingInput/StakingInput.tsx

```diff
@@ -327,6 +333,11 @@ const StakingInput: React.FC<StakingInputProps> = ({
 						</RowTitle>
 						<RowValue>{equivalentSNXAmount}</RowValue>
 					</DataRow>
+					{burnType === BurnActionType.MAX && debtBuffer && setDebtBuffer && (
```

Comment:
do you need to test if `setDebtBuffer` exists here?

Reply:
yes because mint tab and burn tab both share this staking input, only burnTab will be pass the props ```debtBuffer and setDebtBuffer``` so its undefined in some cases

Reply:
awesome